integration_tests/e2e/overview/preInductionOverview.cy.ts

```diff
@@ -65,29 +45,26 @@ context('Prisoner Overview page - Pre Induction', () => {
 
   it('should render prisoner Overview page listing no prisoner goals given prisoner has no goals created pre-induction', () => {
     // Given
-    cy.task('stubSignInAsUserWithEditAuthority')
-    cy.signIn()
-
-    const prisonNumber = 'A00001A'
+    const prisonNumberForPrisonerWithNoGoals = 'A00001A'
```

Comment:
This test uses a different prisoner that is setup with different stub data, hence having to re-setup the stubs